pkg/cache/clusterqueue.go

```diff
@@ -265,18 +270,30 @@ func (c *clusterQueue) inactiveReason() (string, string) {
 			reasons = append(reasons, kueue.ClusterQueueActiveReasonAdmissionCheckInactive)
 			messages = append(messages, fmt.Sprintf("references inactive AdmissionCheck(s): %v", c.inactiveAdmissionChecks))
 		}
+
+		// TBD: This shouldn't be feature gated, as the messages and reasons are valid
```

Comment:
I guess we don't need to feature gate it, because if the gate is disabled thee value `multipleSingleInstanceControllersChecks` would be empty anyway. Right? If this is the case, maybe just comment: "This doesn't need to be gated behind <gate name>, because it is empty when the gate is disabled"